### PR TITLE
Adding manifests for v2.3.0 ChaosCenter

### PR DESCRIPTION
**Pull request by Jonsy13**
## Proposed changes

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

**Review by ishangupta-ds — APPROVED** [review]

**Review by rajdas98 — COMMENTED** [review]
@Jonsy13 should we add server version upgrade init container?

**Review by oumkale — APPROVED** [review]

File: litmus-portal/cluster-k8s-manifest.yml [hunk 1]
```diff
@@ -563,6 +563,31 @@ spec:
...
+            - name: DB_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: litmus-portal-admin-secret
+                  key: DB_PASSWORD
+            - name: DB_SERVER
+              valueFrom:
+                configMapKeyRef:
+                  name: litmus-portal-admin-config
+                  key: DB_SERVER
```
Comment (ishangupta-ds) [inline litmus-portal/cluster-k8s-manifest.yml:590]: can be updated to

```envFrom:
    - configMapRef:
           name: litmus-portal-admin-config

**Review by ishangupta-ds — APPROVED** [review]

**Review by oumkale — APPROVED** [review]

File: litmus-portal/cluster-k8s-manifest.yml (hunk 1 above)
Comment (ishangupta-ds) [reply litmus-portal/cluster-k8s-manifest.yml:590]: @Jonsy13

Comment (Jonsy13) [reply litmus-portal/cluster-k8s-manifest.yml:590]: We are taking values from secret as well as configMap.

Comment (ishangupta-ds) [reply litmus-portal/cluster-k8s-manifest.yml:590]: then we can add
```
- secretRef:
            name: litmus-portal-admin-secret
```

Comment (ishangupta-ds) [reply litmus-portal/cluster-k8s-manifest.yml:590]: @Jonsy13 

```
envFrom:
    - configMapRef:
           name: litmus-portal-admin-config
    - secretRef:
           name: litmus-portal-admin-secret
```

Comment (Jonsy13) [reply litmus-portal/cluster-k8s-manifest.yml:590]: @rajdas98 Do we need all secrets to be passed to initContainer?

**Review by rajdas98 — APPROVED** [review]

**Pull request merged**